### PR TITLE
DOCS-4107: TC 2.2.1 relnotes

### PR DESCRIPTION
**Pull request by Mrt134**
Additional - switch from html tags to markdown unordered list for tickets in the 2.2 changelog
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/TrueCommand/TCReleaseNotes.md [hunk 1]
```diff
@@ -23,8 +23,30 @@ aliases:
 | 2.3.0 | Tag | TBD |
 | 2.3.0 | Release | TBD |
 
+## 2.2.1 Hotpatch
```
Comment (william-gr) [inline content/TrueCommand/TCReleaseNotes.md:26]: We have been saying "hotpatch" internally, but I am not sure thats the wording we want to use

Its just another dot release

Comment (Mrt134) [reply content/TrueCommand/TCReleaseNotes.md]: fixed!

File: content/TrueCommand/TCReleaseNotes.md [hunk 2]
```diff
@@ -23,8 +23,30 @@ aliases:
 | 2.3.0 | Tag | TBD |
 | 2.3.0 | Release | TBD |
 
+## 2.2.1
+
+**August 12, 2022**
+
+This is a small hotpatch designed to address some additional cluster issues found between TrueCommand 2.2 and SCALE 22.02.3.
```
Comment (william-gr) [inline content/TrueCommand/TCReleaseNotes.md:30]: This is a minor patch release designed... maybe?

**Review by ericbsd — APPROVED** [review]
LGTM

**Pull request merged**